src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -3966,6 +4027,41 @@ export class StudyViewPageStore
         this.changeChartVisibility(uniqueKey, false);
     }
 
+    public getGenePanelDataCount(
+        chartMeta: ChartMeta
+    ): MobxPromise<ClinicalDataCountSummary[]> {
+        const suffix = chartMetaToProfile(chartMeta);
+
+        return remoteData<ClinicalDataCountSummary[]>({
```

Comment:
this is gonna produce a new remoteData every time the method is called, which is a potential memory leak.  also, it isn't actually fetching anything remote (that's done by genePanelCounts.  We need to cache this promise (see getGenomicChartDataBin)